app/models/user.rb

```diff
@@ -8,8 +8,7 @@ class User < ApplicationRecord
 
   devise :database_authenticatable if Rails.env.development? && ENV["2FA_ENABLED"] != "true"
 
-  has_many :decisions, dependent: :restrict_with_exception
-  has_many :planning_applications, through: :decisions
```

Comment:
Just spotted this while I was trying out different ways of searching. The `decisions` table no longer exists, and `planning_applications` now has `user_id`.